js/interface.js

```diff
@@ -214,7 +214,9 @@ var DynamicLists = (function() {
           item.id = _this.makeid(8);
           item.column = 'none';
           item.logic = 'none';
+          item.valueType = 'enter-value';
           item.value = '';
+          item.valueField = 'none';
```

Comment:
Why is it `none`? If the `valueType` is `enter-value` shouldn't `valueField` be `Value`?

```suggestion
          item.valueField = 'Value';
```

Reply:
It doesn't matter because they always checks which value contains in valueType but i can change valueField default value to 'Value'

Reply:
@AndrRyaz Go ahead. You don't need to wait for another approval when I already gave my perspective 3 days ago. It creates unnecessary delay. cc @sofiiakvasnevska

Reply:
@tonytlwu ok, sorry, fixed